### PR TITLE
Adds imagePickerIOS component bindings from Rebolt.

### PR DESCRIPTION
**Pull request by Trancever**
Brings imagePickerIOS component bindings from Rebolt.

### CODE REVIEW

File: src/imagePickerIOS.rei [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+let canRecordVideos: (unit => unit) => unit;
+
+type error = {
+  .
+  "code": int,
+  "message": string,
+};
+
+let openCameraDialog: (~videoMode: bool, unit => unit, error => unit) => unit;
```
Comment (wokalski) [inline src/imagePickerIOS.rei]: I think all these args should be labelled. WDYT?

Comment (Trancever) [reply src/imagePickerIOS.rei]: Yeah, that's good idea. Are onSuccess and onError good labels?

Comment (wokalski) [reply src/imagePickerIOS.rei]: yes! It'd be cool do it for both open* functions

Comment (Trancever) [reply src/imagePickerIOS.rei]: Yeah, it should be done right now.

**Comment by wokalski** [comment]
Nice, thanks.

**Pull request merged**